data/build_probe_data.py

```diff
@@ -0,0 +1,120 @@
+import tensorflow.compat.v1 as tf
+from tqdm import tqdm
+from collections import defaultdict
+import sklearn
+import numpy as np
+import json
+
+with tf.io.gfile.GFile("gs://e2e_central/data/ml-sequences-train.tsv", 'r') as f:
```

Comment:
instead of hard-coding the filenames, please have filenames configurable through flags.
you can set the default values of flags to those values.